### PR TITLE
[REVIEW FIRST] Postgres Support

### PR DESCRIPTION
**Pull request by zysim**
Closes: #4 

Adds:
- Reading things from `.env`
- Registering `DbContext`s to use Postgres

Note: Our Postgres config is loaded via `.env` instead of using secrets or JSONs. This is because `docker-compose.yml` reads from `.env` by default, so we'd might as well source that same file for our config.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -7,3 +7,8 @@ end_of_line = lf
 insert_final_newline = true
 indent_style = tab
 charset = utf-8
+
+# docker-compose.yml doesn't like tabs lmao
+[.yml]
+indent_style = space
+indent_size = 4
```
Comment (RageCage64) [inline .editorconfig:14]: Some people want yaml to be 2 spaces, I don't think it matters much to me though and it's easy enough to change

**Review by RageCage64 — APPROVED** [review]
Your changes all look good to me @zysim so if you're good with mine, we can merge this in.

File: .editorconfig (hunk 1 above)
Comment (zysim) [reply .editorconfig:14]: ¯\\_(ツ)\_/¯ if anyone else wants to change this really inconsequential thing later in the future they can. It's not worth tacking on another commit just to change a number.

**Comment by zysim** [comment]
Fixed a type cast. Merging this in. :tada:

**Pull request merged**